2 дз/2 дз/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿//Geometry
+using System;
+
+namespace Geometry
+{
+    class Program
+    {
+        static double area(double edge)
+        {
+            return (2 * (Math.Sqrt(3)) * (edge * edge));
+        }
+
+        static double volume(double edge)
```

Comment:
В C# методы принято называть с заглавной буквы.
```suggestion
        static double Volume(double edge)
```